sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuDataWritingCommandExec.scala

```diff
@@ -116,25 +116,40 @@ case class GpuDataWritingCommandExec(cmd: GpuDataWritingCommand, child: SparkPla
   private lazy val sideEffectResult: Seq[ColumnarBatch] =
     cmd.runColumnar(sparkSession, child)
```

Comment:
Arguably `GpuDataWritingCommand.runColumnar` was a mistake.  No data writing commands return columnar data, nor likely ever will.  We should just invoke the `run` method on these and avoid the ColumnarBatch mess (and rip out `GpuDataWritingCommand.runColumnar`).

Reply:
Updated